server/game/cards/23-AHaH/Sweetrobin.js

```diff
@@ -5,7 +5,7 @@ class Sweetrobin extends DrawCard {
     setupCardAbilities(ability) {
         this.interrupt({
             when: {
-                onCardRevealed: event => event.card.hasPrintedCost() && event.card.getPrintedCost() <= event.card.owner.getTotalInitiative() && ['hand', 'draw deck', 'shadows'].includes(event.card.location)
+                onCardRevealed: event => (!event.card.hasPrintedCost() || (event.card.hasPrintedCost() && event.card.getPrintedCost() <= event.card.owner.getTotalInitiative())) && ['hand', 'draw deck', 'shadows'].includes(event.card.location)
```

Comment:
As per my comment [here](https://github.com/throneteki/throneteki/pull/3372#issuecomment-1588256333), this needs to be reverted. May need to adjust your WotE test to compensate in some way. :)

Reply:
omg, totally wrong with that :D You're right, I thought that "X" cost and "-" cost were the same for the purpose of determining the value. Reverted change